salt/modules/keystone.py

```diff
@@ -699,7 +705,16 @@ def user_get(user_id=None, name=None, profile=None, **connection_args):
                 break
     if not user_id:
         return {'Error': 'Unable to resolve user id'}
-    user = kstone.users.get(user_id)
+    try:
+        user = kstone.users.get(user_id)
+    except keystoneclient.exceptions.NotFound:
+        log.error(
+            'Could not find user \'{0}\''.format(
+                user_id
+            )
+        )
+        return ret
```

Comment:
Above we're handling an error by returning a dict with an error message and here we're just logging it. Should these be consistent?